signup.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel='stylesheet' href='https://breezecss.netlify.app/styles/breeze.css' >
+    <link rel="stylesheet" href="./styles/main.css" class="stylesheet">
+    <title>ipops</title>
+</head>
+<body>
+    <header class="navbars">
+        <!--a href="#" class="menu-icon">
+            <img src="path-of-the-image" width="35px" height="35px" alt="logo" />
+        </!--a>-->
+        <a href="" class="brand-name h6">
+            ipops
+        </a>
+        <nav>
+            <ul>
+                <li class="list-items">
+                    <i class="fas fa-heart fa-lg"></i>
+                </li>
+                <li class="list-items">
+                    <i class="fas fa-shopping-cart fa-lg"></i>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <!-----login section---->
+    <main>
+    <div class="form-container">
+        <form class="form">
+            <h1 class="form-heading h5">Sign Up</h1>
+            <div class="input-grp">
+                <label>Name</label>
+                <input type="text" class="input-field" />
+            </div>
+            <div class="input-grp">
+                <label>Email</label>
```

Comment:
here for the label you can give for="email" and then input an id of "email" to associate both so that in case of screen readers when focus comes on form input it reads for the label like what is the use of that input here.

Reply:
okay, will make the changes.